src/proc/pointcloud.cpp

```diff
@@ -18,7 +18,7 @@
 #include "../cuda/cuda-pointcloud.cuh"
 #endif
 #ifdef __SSSE3__
```

Comment:
What happens when both RS2_USE_CUDA and __SSSE3__  defines are set, is CUDA prioritized over SSSE ?